apps/judicial-system/backend/src/app/modules/defendant/test/civilClaimantController/create.spec.ts

```
@@ -0,0 +1,80 @@
+
+    mockCivilClaimantModel = civilClaimantModel
+
+    const mockCreate = mockCivilClaimantModel.create as jest.Mock
+    mockCreate.mockResolvedValue(createdCivilClaimant)
+
+    givenWhenThen = async () => {
+      const then = {} as Then
+
+      await civilClaimantController
+        .create(theCase.id, theCase)
+        .then((result) => (then.result = result))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)